### PR TITLE
average pool time ping thing

### PR DESCRIPTION
**Pull request by memedmusic**
no bugs!!!!

### CODE REVIEW

**Comment by TheBlueBurger** [comment]
I'll review this when I get home, it's impossible nothing broke

File: src/mining.js [hunk 1]
```diff
@@ -328,16 +328,21 @@ async function selectPool(minerData, algo) {
 				let spinner = ora("Calculating ping").start();
 				let pings = [];
 				for await(regionToTest of regionList) {
-					let domain = poolsy.algos[algo].host.split("://")[1].split(":")[0].replace("REGION", regionToTest.value);
-					let timeStarted = Date.now();
-					try {
-					await fetch("http://" + domain);
-					} catch {
+					let i;
+					var totalPing = 0;
```
Comment (TheBlueBurger) [inline src/mining.js:332]: You don't need this, you can define them in the loop

File: src/mining.js [hunk 2]
```diff
@@ -328,16 +328,21 @@ async function selectPool(minerData, algo) {
...
+					for(i=0;i<5;i++){
+						let domain = poolsy.algos[algo].host.split("://")[1].split(":")[0].replace("REGION", regionToTest.value);
+						let timeStarted = Date.now();
+						try {
+							await fetch("http://" + domain);
+						} catch {
 						// i dont care, still sent the request
+						}
+						totalPing = totalPing + Date.now() - timeStarted
```
Comment (TheBlueBurger) [inline src/mining.js:341]: You can use +=

**Review by TheBlueBurger — CHANGES_REQUESTED** [review]

File: src/mining.js (hunk 1 above)
Comment (memedmusic) [reply src/mining.js:332]: let i, yes
var totalPing, I don't think that works, an undefined error will occur on line 341

Comment (TheBlueBurger) [reply src/mining.js:332]: I meant i, github for android is broken

Comment (memedmusic) [reply src/mining.js:332]: ahh lol

**Comment by TheBlueBurger** [comment]
I'll try running this when I come home, should be tested within 7 hours if I'm not lazy

**Review by Vukkyy — APPROVED** [review]
seems ok. to me. waiting for burgers blue.

**Review by TheBlueBurger — APPROVED** [review]
Hopefully this doesn't break everything

**Comment by TheBlueBurger** [comment]
@memedmusic @Vukkyy is there any problems with this PR or can I merge? I can't find any right now

**Comment by TheBlueBurger** [comment]
I'll merge this now

**Pull request merged**